src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -375,6 +386,16 @@ export default (accessToken, userID, opts) => {
     ),
     matchUsersLoader: gravityLoader("match/users", {}, { headers: true }),
     matchSetsLoader: gravityLoader("match/sets", {}, { headers: true }),
+    authenticatedHeroUnitsLoader: gravityLoader(
```

Comment:
My only hesitation is this case:

* you are signed into force with an admin account
* you hit the homepage
* you get ALL hero units rather than just the ones that are included in the `active` scope

I DO want all of them when the query comes from forque, but not from force. Do we have a good pattern for this? I was going to use a flag but then we were thinking I could just see which loaders are available. That doesn't really work for this case!